public/data/Priyanshu.json

```
@@ -0,0 +1,12 @@
+{
+  "name": "Priyanshu Varshney",
+  "location": "India",
+  "bio": "A Web Programmer",
+  "avatar": "https://avatars.githubusercontent.com/u/124420199?s=400&u=fceacb048a62263a94621c775c13754b8dbc9bd0&v=4"
```

```suggestion
  "avatar": "https://avatars.githubusercontent.com/u/124420199?s=400&u=fceacb048a62263a94621c775c13754b8dbc9bd0&v=4",
```
Add comma at last after avatar tag.